cisagov/manage.get.gov
# Issue #2406: Clipboard Functionality [backup]

## Ticket 2406

Resolves #2406


## Changes


- change_form_object_tools.html (added a button that copies summary to clipboard if a url is not provided)
- copy-summary.js (added function to copy summary to clipboard)
- contact_detail_list.html (added ids for safer DOM traversing)



## Context for reviewers



## Setup



## Code Review Verification Steps

code is deployed to [sandbox-rh](https://getgov-rh.app.cloud.gov/admin/registrar/domainrequest/163/change/?_changelist_filters=status__in%3Dsubmitted%252Cin%2Breview%252Caction%2Bneeded#)

- Navigate to /admin
- Open a domain request
- Verify that...
   - [ ] a button appears in the upper right that says "Copy to clipboard".  
   - [ ] Clicking it copies a summary of the Domain Request to clipboard in a format that matches the ticket ACs

### As the original developer, I have

#### Satisfied acceptance criteria and met development standards


#### Ensured code standards are met (Original Developer)


#### Validated user-facing changes (if applicable)


### As a code reviewer, I have

#### Reviewed, tested, and left feedback about the changes


#### Ensured code standards are met (Code reviewer)


#### Validated user-facing changes as a developer




**Note:** Multiple code reviewers can share the checklists above, a second reviewers should not make a duplicate checklist

### As a designer reviewer, I have

#### Verified that the changes match the design intention


#### Validated user-facing changes as a designer




## Screenshots
